docs/joss/paper.bib

```diff
@@ -16,15 +16,15 @@ @misc{g95-site
 
 @misc{ifx-site,
   author={{Intel Corporation}},
-  title = {{Intel Fortran Compiler}},
+  title = {Intel {F}ortran Compiler},
   journal = {Web page},
   year = 2025,
   url = {https://www.intel.com/content/www/us/en/developer/tools/oneapi/fortran-compiler.html}
 }
 
 @misc{cce-site,
   author={{Hewlett Packard Enterprise}},
-  title = {{Cray Compiling Environment Fortran compiler}},
+  title = {Cray Compiling Environment {F}ortran compiler}},
```

Comment:
For example, just as "Fortran" is the capitalized proper name of a programming language, the Cray Compiling Environment is the proper name of an HPE product which is always capitalized, even outside of titles, when it appears in [company literature](https://cpe.ext.hpe.com/docs/latest/cce/index.html). 

```suggestion
  title = {{Cray Compiling Environment Fortran compiler}},
```